test/commands/shell.js

```diff
@@ -23,12 +23,23 @@ function stream(command, location) {
 
   child.stdout.pipe(process.stdout);
   child.stderr.pipe(process.stderr);
+
+  return child;
 }
 
 var commands = {};
 
 commands.cat = function cat(file) {
-  return stream('cat', file);
+  var proc = stream('cat', file);
+
+  proc.stderr.once('data', function(data) {
+    if (/No such file or directory/.test(data.toString())) {
+      console.log(data.toString());
```

Comment:
Current adb emits this on stdout, which is why I also have it going there. If you look at routes/files.js (currently line 41), you'll see that real adb for some reason pushes this on stdout, and I commented it as such as well.